packages/@lwc/integration-karma/test/template/attribute-class/object-values.spec.js

```diff
@@ -37,6 +37,21 @@ function testReactiveClassNameValue(name, setupFn, updateFn, expected) {
     });
 }
 
+/** Stub of LBC's `classSet`. Has enumerable keys on the prototype. */
+function classSet(props) {
+    const proto = {
+        add() {},
+        invert() {},
+        toString() {
+            return Object.entries(props)
+                .filter(([, val]) => val)
+                .map(([key]) => key)
+                .join(' ');
+        },
+    };
+    return Object.assign(Object.create(proto), props);
```

Comment:
Let's also add something like:

```js
Object.defineProperty(result, 'bar', {
  value: 'bar',
  enumerable: false
})
Object.defineProperty(result, 'baz', {
  value: 'baz',
  enumerable: true
})
```

to confirm that only `enumerated` props are ignored.